.eslintrc

```diff
@@ -1,17 +1,11 @@
 {
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
-  },
+  "extends": "eslint-config-airbnb",
   "env": {
     "browser": true,
     "mocha": true,
     "node": true
   },
-  "parser": "babel-eslint",
```

Comment:
Isn't it already defined [here](https://github.com/airbnb/javascript/blob/master/linters/.eslintrc#L2) ?